### DISCUSSION TITLE
how adding script and styles in component content?

### DISCUSSION THREAD

**Discussion by gitwithsarthak** [GrapesJS/grapesjs#3973] [Q&A] [answered]
Hi, @artf

Like this type of link and script tags and external links how we can add in a component content section

  `<link rel="stylesheet" href="/theme/HTML/css/bootstrap.css" type="text/css" />`
           `<link rel="stylesheet" href="/theme/HTML/style.css" type="text/css" />`
         ` <link rel="stylesheet" href="/theme/HTML/css/dark.css" type="text/css" />`
      `    <script src="/HTML/js/functions.js"></script>`

**Comment by Ju99ernaut** [comment, marked as answer]
https://grapesjs.com/docs/modules/Components-js.html#dependencies not sure if this is your exact use case.

**Reply by gitwithsarthak** [reply]
we have many styles and scripts. so each link have been defined in separated brackets [ ] or all in one [ ' ', ' ', ' ' ] 

                       ['<link rel="stylesheet" href="http://localhost:3000/assets/dark.css" type="text/css" />'],
                       ['<link rel="stylesheet" href="http://localhost:3000/assets/css/fonts.css" type="text/css" />'],

**Reply by gitwithsarthak** [reply]
Done removing link tags

canvas:{
    styles:[
       'http://localhost:3000/assets/css/headers/style.css',
       'http://localhost:3000/assets/css/headers/dark.css',
   ]
 }
